automatic/freecad/update_helper.ps1

```diff
@@ -46,6 +46,11 @@ function Get-FreeCad {
     [version]$version = ( ( ($version.Major), ($version.Minor), ($filler), ($version.Build) ) -join "." )
   }
   $vert = @{$true = "$version-$kind"; $false = "$version" }[( $kind -eq 'dev' )]
+# The url64 should always be valid
+$test = (Update-OnETagChanged -execUrl "${PreUrl}${url64}" -saveFile "${PSScriptRoot}\freecad.info" -kind $kind -OnETagChanged { @{} } -OnUpdated { @{} } -boule )
```

Comment:
Instead of updating the helper, two separate info files should have been used instead.

The helper is very simple on purpose, without having knowledge of streams/kinds.